README.md

```diff
@@ -1 +1,79 @@
-Keep your branches clean and nice. Use flake8 and tests. Good luck!
\ No newline at end of file
+## Project description
+This web application allows you to create a local and public IP entry, view and email a list of data
+
+## Quick start
+### First step: 
+- Copy project from git
+```
+git clone git clone https://github.com/nester256/rpm-hws_7-1_2023/tree/nesterov
+
+cd rpm-hws_7-1_2023
+
+git checkout nesterov
+```
+### Second step:
+- Create docker image 
+```
+docker run  -d \
+        --name ips_test \
+        -e POSTGRES_USER=admin \
+        -e POSTGRES_PASSWORD=1234 \
+        -e PGDATA=/postgres_data_inside_container \
+        -v ~/sirius_db_2023/postgres_data:/postgres_data_inside_container \
+        -p 38746:5432 \
+        postgres:15.1
+```
+### Third step:
+- Create tables
+```
+psql -h 127.0.0.1 -p 5435 -u admin ips_tests -f db_init.ddl
```

Comment:
flag '-u' is not a thing

Reply:
I never use this command, so I don't know how to write it correctly

Reply:
![image](https://user-images.githubusercontent.com/113025394/231017063-f9e0b5d3-df4c-4c85-88b1-5912d90d2844.png)
I specify the user I correctly?

Reply:
it is either `-U` or `--username`, definetely not `-u`
don't u see the difference?

moreover, it is not important whether u use it or not - u put it in ur project readme, so you are the one responsible now.